### PR TITLE
[fs] Layout migration

### PR DESCRIPTION
**Pull request by photodow**
### Related Ticket(s)

#1001 

### Changelog

**Changed**

- migrated layout functional spec content over.

### CODE REVIEW

File: wiki/Layout-component/Layout.md [hunk 1]
```diff
@@ -39,4 +37,128 @@ The Layout core component helps us to layout our different components with a mor
...
+
+<br />[Back to top](#wiki-wrapper)<br /><br /><br />
+
+
+
+### 4. Border
+
+`optional` `boolean`
+
+The optional border is displayed at the bottom of the [[Layout]] providing a visual indicator for a change of topic.Typically used when nesting the component.
```
Comment (oliviaflory) [inline wiki/Layout-component/Layout.md:121]:
```suggestion
The optional border is displayed at the bottom of the [[Layout]] providing a visual indicator for a change of topic. Typically used when nesting the component.
```

File: wiki/Layout-component/Layout.md [hunk 2]
```diff
@@ -39,4 +37,128 @@ The Layout core component helps us to layout our different components with a mor
...
+
+###### Logic
+
+|     | condition          | description | |
+|:----|:-------------------|:------------|:-|
+| 3.1 | `border === true`  | If border is `true` then the pattern’s border at the bottom should render. |  |
```
Comment (oliviaflory) [inline wiki/Layout-component/Layout.md:127]: Do we want to reference "pattern" here? I'd guess this is carry over from when we called layout component's patterns?

```suggestion
| 3.1 | `border === true`  | If border is `true` then the layout component's border at the bottom should render. |  |
```

File: wiki/Layout-component/Layout.md [hunk 3]
```diff
@@ -39,4 +37,128 @@ The Layout core component helps us to layout our different components with a mor
...
+| 3.2 | `border === false` | If border is `false` then the pattern’s border should NOT render. | `default` |
```
Comment (oliviaflory) [inline wiki/Layout-component/Layout.md:128]:
```suggestion
| 3.2 | `border === false` | If border is `false` then the layout component's border should NOT render. | `default` |
```

File: wiki/Layout-component/Layout.md [hunk 4]
```diff
@@ -39,4 +37,128 @@ The Layout core component helps us to layout our different components with a mor
...
+<!-->
+    Note that `4-4-4-4` layout is not an option at this time.
+
+    Also, notice how each column has a corresponding child.
+    You could also optionally leave off a column from right
+    to left if you don’t need it for a given layout.
+ </-->
+```
+
+> ![image](https://user-images.githubusercontent.com/3793636/117873641-a6835d00-b265-11eb-8433-8c9c73a2e999.png) Note the code examples above maybe react specific, but the functionality should stand across any language with how it relates to being built out.
```
Comment (oliviaflory) [inline wiki/Layout-component/Layout.md:90]: As a non-dev I might suggest changing "across any language" to "across any framework", when I first read it I thought of language = translating to different geographies.

```suggestion
> ![image](https://user-images.githubusercontent.com/3793636/117873641-a6835d00-b265-11eb-8433-8c9c73a2e999.png) Note the code examples above maybe react specific, but the functionality should stand across any framework with how it relates to being built out.
```

**Review by oliviaflory — COMMENTED** [review]
Added some suggestions

File: wiki/Layout-component/Layout.md (hunk 2 above)
Comment (photodow) [reply wiki/Layout-component/Layout.md:127]: ah, great catch. I've been trying to get all of these old pattern references.

File: wiki/Layout-component/Layout.md (hunk 4 above)
Comment (photodow) [reply wiki/Layout-component/Layout.md]: I just removed the whole thing. Confusing, and blurring the line between technical documentation.

**Review by oliviaflory — APPROVED** [review]
LGTM 👍

**Pull request merged**